test/studies/lulesh/bradc/xyztuple/lulesh-dense-3tuple.chpl

```diff
@@ -272,11 +272,13 @@ proc main() {
   if printCoords {
     var outfile = open("coords.out", iomode.cw);
     var writer = outfile.writer();
-    var fmtstr = if debug then "%1.9e" else "%1.4e";
+    var fmtstrnum = if debug then "%1.9er" else "%1.4er";
+    var fmtstr = fmtstrnum + " " + fmtstrnum + " " + fmtstrnum + "\n";
     for i in Nodes {
-      writer.writeln(format(fmtstr, xyz[i][X]), " ", 
-                     format(fmtstr, xyz[i][Y]), " ", 
-                     format(fmtstr, xyz[i][Z]));
+      //
+      // TODO: Why doesn't this one work?
+      //
+      writer.writef(fmtstr, xyz[i][X], xyz[i][Y], xyz[i][Z]);
```

Comment:
I think this dated back to when I was struggling with C vs. Chapel format strings on that plane flight a month or so back before you set me straight.